src/world/area_arn/arn_05/arn_05_3_npc.c

```diff
@@ -441,32 +441,32 @@ EvtScript N(EVS_Scene_TubbaRaid) = {
     EVT_CALL(ShowEmote, NPC_Boo_02, EMOTE_EXCLAMATION, 45, 20, EMOTER_NPC, 0, 0, 0, 0)
     EVT_THREAD
         EVT_CALL(PlaySound, SOUND_BOO_APPEAR)
-        EVT_WAIT(20)
+        EVT_WAIT(20 * DT)
         EVT_SETF(LVar0, EVT_FLOAT(240.0))
-        EVT_LOOP(20)
-            EVT_SUBF(LVar0, EVT_FLOAT(12.0))
+        EVT_LOOP(20 * DT)
+            EVT_SUBF(LVar0, EVT_FLOAT(240.0 / (s32) (20 * DT)))
```

Comment:
Does this math work for the other Boo fading increments from the last PR? If so it would be nice to apply that there instead of using #defines for the values.